internal/cmd/root/help.go

```diff
@@ -0,0 +1,129 @@
+package root
```

Comment:
Was this meant to be in its own `help` package?

Reply:
root in this case is correct. It's also a common pattern since this is basically the default command to run when it doesn't recognize anything else.